src/radical/pilot/staging_directives.py

```diff
@@ -199,10 +184,6 @@ def complete_url(path, context, log=None):
         log.debug('   expand with %s', context[schema])
         ret = ru.Url(context[schema])
 
-        if schema in ['resource', 'pilot']:
-            # use a dedicated staging area dir
-            ret.path += '/staging_area'
-
```

Comment:
Does this mean that the pilot sandbox gets renamed? Is it not `session_id/pilot.0000/sandbox` anymore? This is something that goes all over the PR.

Reply:
Oh no - the pilot sandbox remains as it is.  The special `staging_area/` subdir within the pilot sandbox gets removed though.

Reply:
And any data are moved to the same folder with all the log files?

Reply:
Yes, indeed.  I realize that this is not optimal, and I suggest to succesively move log, prof and cfg/json files out of the pilot sandbox.  Ideally, the user should be able to inspect the sandbox and mostly recognize workload related data and directories.  But the motivation for this specific change was that it turned out to be difficult to construct relative paths in relation to the `staging_area/`, and the change simplifies sandbox references a lot: `pilot://` refers to the pilot sandbox, `unit://` to the unit sandbox, etc.  This in turn simplifies data management for several projects (scalems, repex, covid).